library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -402,6 +410,12 @@ public int read(ExtractorInput input, PositionHolder seekPosition) throws IOExce
       case ID_CONTENT_ENCODING:
         // TODO: check and fail if more than one content encoding is present.
         return;
+      case ID_CONTENT_COMPRESSION:
+        // A number of muxers (e.g. mkvmerge v4.0.0) do not write EBML elements which correspond to the default values
+        // as defined in the spec. They assume that if they provide a ID_CONTENT_COMPRESSION with no specific
+        // ID_CONTENT_COMPRESSION_ALGORITHM, zlib will be used.
+        currentTrack.contentCompression = CONTENT_COMPRESSION_ZLIB;
```

Comment:
instead of doing this, you can add a hasContentCompression boolean and set it to true here. contentCompression can then default to 0 rather than being explicitly set here (since that's the default according to the spec anyway [1]).

[1] https://matroska.org/technical/specs/index.html#ContentCompAlgo